app/tags/page.tsx

```diff
@@ -0,0 +1,44 @@
+import Link from '@/components/Link'
```

Comment:
Is this page linked anywhere?

Reply:
No, this is a hidden page. I'm not a fan of the tag bubble, honestly, but there is a `ListLayoutWithTags` layout that was provided in v2 which adds a small sidebar with a list of tags on `/blog`. It looked cluttered when I used it for the first time but maybe I can try again!